doc/source/conf.py

```diff
@@ -0,0 +1,227 @@
+# -*- coding: utf-8 -*-
+#
+# SimPhoNy documentation build configuration file, created by
+# sphinx-quickstart on Mon Feb 02 16:26:16 2015.
+#
+# This file is execfile()d with the current directory set to its
+# containing dir.
+#
+# Note that not all possible configuration values are present in this
+# autogenerated file.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+
+import sphinx.environment
+from docutils.utils import get_source_line
+
+def _warn_node(self, msg, node, **kwargs):
+    if not msg.startswith('nonlocal image URI found:'):
+        self._warnfunc(msg, '%s:%s' % get_source_line(node), **kwargs)
+
+
+sphinx.environment.BuildEnvironment.warn_node = _warn_node
+
+def mock_modules():
+    import sys
+
+    from mock import MagicMock
+
+    try:
```

Comment:
This one and the other try are technically not needed. I would comment one out as an example (comment it as such) and remove the other